cards/ecg-right-posterior-leads/card.md

```diff
@@ -52,3 +52,6 @@ Hints of an associated posterior infarct:
 2.  Upright T wave waves (in later stage)
 
 ## References
+
+1. [Edhouse J, Brady WJ, Morris F. ABC of clinical electrocardiography: Acute myocardial infarction-Part II. BMJ. 2002; 324: 963-6.] (https://doi.org/10.1136/bmj.324.7343.963)
```

Comment:
Like other review, this should be unordered list.

We are using a new routine in the build process that enforces some markdown formatting.

Looks like it requires a <tab> after the dash for lists, eg:

```
Here is a list

-    first item
-    second item
-    third item
```
Note tab spacing after the dash. Also note the blank line before the list.